docs/trex_sandbox_test.md

```diff
@@ -22,7 +22,7 @@ Tableau supports two types of dashboard extensions:
 
 * **Network Enabled Extensions** - can access resources and applications outside of Tableau. Supported in Tableau 2018.2 and later.
 
-* **Sandboxed Extensions** - run in a Tableau hosted environment and cannot make network calls. Supported in Tableau 2019.4 and later. Available for testing with Tableau 2019.3.
+* **Sandboxed Extensions** - run in a Tableau hosted environment and cannot make network calls. Supported in Tableau 2019.4 and later.
```

Comment:
can we / do we remove this after 2019.4 is no longer in the support window?

Reply:
I'll add that to my future notes list. In this pass I left these mostly alone.

Reply:
We no longer support 2019.4 (see supported versions below)
https://www.tableau.com/support/services

So in the next pass, we'll remove any comments about these versions?

Reply:
I understand that - what I meant was I didn't want to tackle that here in this editing pass. There are a lot of versions mentioned everywhere so we'll need to figure out the strategy for where/when to update them.

Reply:
I included this on my "future doc improvements" list in #464

Reply:
I agree.  It's kind of a mess. As new features have been introduced or as other factors changed (looking at you Chrome and Qt), we've had to kind of layer the versions in.
 I've had a work item to add a topic about versions and compatibility. That would probably help.